arangodb-net-standard/CollectionApi/GetCollectionResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Net;
+
+namespace ArangoDBNetStandard.CollectionApi
+{
+    public class GetCollectionResponse
+    {
+        public bool Error { get; set; }
+        public HttpStatusCode Code { get; set; }
+        public int Type { get; set; }
+        public bool IsSystem { get; set; }
+        public string GloballyUniqueId { get; set; }
+        public string Id { get; set; }
+        public string Name { get; set; }
+        public int Status { get; set; }
+    }
+}
```

Comment:
We should end a file with a line break, git warns about this, there are some reasons: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
I always wondered about the reasons, today I learned something 😄